base/set.jl

```diff
@@ -191,6 +191,62 @@ function unique(f::Callable, C)
 end
 
 """
+    unique!(c)
```

Comment:
I guess `A` or `a` would make more sense as long as this only applies to vectors.